### PR TITLE
added CI configurations from operate-first/apps

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
In response to WG-Devsecops discussion today: CI changes were to factored out of initial ADR structure pr.

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
depends on either `ok-to-test` label or [community pr 101](https://github.com/open-services-group/community/pull/101)

**Review by harshad16 — COMMENTED** [review]
/ok-to-test
lgtm

File: .prow.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-              requests:
-                memory: "500Mi"
-                cpu: "300m"
-              limits:
-                memory: "1Gi"
-                cpu: "300m"
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: aicoe-ci/prow/pre-commit
```
Comment (harshad16) [inline .prow.yaml]:
```suggestion
    context: pre-commit
```

Comment (Gregory-Pereira) [reply .prow.yaml]: whoops haha, resolving now

**Pull request merged**